prow/plugins.yaml

```diff
@@ -72,112 +69,64 @@ approve:
   implicit_self_approve: true
 
 plugins:
-  istio/istio:
-  - approve
-  - assign
-  - blunderbuss
-  - golint
-  - help
-  - hold
-  - lgtm
-  - lifecycle
-  - slackevents
-  - trigger
```

Comment:
we can't rerun prow tests without this

Reply:
This is only moved in this file, the functionality is not disabled for the repo.

Reply:
Oops, I missed it since its defined in `istio` and not `istio/istio`.

Reply:
The istio plugins do not apply to istio/istio. So this PR right does in fact turn off testing in most places.

Ideally we should configure everything at the org level to keep things simpler and less redundant.